core/src/androidTest/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/helpers/MotionEventPrimitive.kt

```diff
@@ -0,0 +1,21 @@
+package com.pierfrancescosoffritti.androidyoutubeplayer.core.helpers
+
+/**
+ * This class is needed to export MotionEvent attributes from Activity to the TestClasses.
```

Comment:
Since all motionEvents point to the same object, they all had the value that was set last. So, I used this class to transfer the values to a new instance.

I found a better way to do that: `MotionEvent.obtain()`